### PR TITLE
Types PR

### PR DESCRIPTION
**Pull request by oddcoder**

### CODE REVIEW

File: libr/core/cmd_type.c [hunk 1]
```diff
@@ -307,14 +287,17 @@ static int cmd_type(void *data, const char *input) {
 			if (*name) {
 				SdbKv *kv;
 				SdbListIter *iter;
-				int tmp_len = strlen (name);
-				char *tmp = malloc (tmp_len + 2);
+				char*type =sdb_const_get (core->anal->sdb_types, name, 0);
```
Comment (XVilka) [inline libr/core/cmd_type.c]: pls fix spaces here

File: libr/core/cmd_type.c [hunk 2]
```diff
@@ -307,14 +287,17 @@ static int cmd_type(void *data, const char *input) {
...
+				if(!type)
+					break;
+				int tmp_len = strlen (name)+strlen(type);
```
Comment (XVilka) [inline libr/core/cmd_type.c]: spaces

File: libr/core/cmd_type.c [hunk 3]
```diff
@@ -307,14 +287,17 @@ static int cmd_type(void *data, const char *input) {
 			if (*name) {
 				SdbKv *kv;
 				SdbListIter *iter;
-				int tmp_len = strlen (name);
-				char *tmp = malloc (tmp_len + 2);
+				char*type =sdb_const_get (core->anal->sdb_types, name, 0);
+				if(!type)
```
Comment (XVilka) [inline libr/core/cmd_type.c]: space

File: libr/core/cmd_type.c [hunk 4]
```diff
@@ -46,49 +46,8 @@ static int sdbdelete(void *p, const char *k, const char *v) {
 	return 1;
 }
 static int typelist(void *p, const char *k, const char *v) {
-#define DB core->anal->sdb_types
 	RCore *core = (RCore *)p;
-	int i;
```
Comment (radare) [inline libr/core/cmd_type.c]: do not remove this code, wrap it with #if 0 .. #endif 
it can be useful at some point.

**Comment by radare** [comment]
Fixed and merged!

**Pull request closed without merging**